react-app/src/features/forms/waiver/contracting/Amendment.tsx

```diff
@@ -122,7 +120,7 @@ export const AmendmentForm = () => {
       attachments={{
         faqLink: "/faq/waiverb-attachments",
       }}
-      defaultValues={{ id: "" }}
+      defaultValues={{ id: "", waiverNumber: waiverId }}
```

Comment:
```suggestion
      defaultValues={{ id: "", waiverNumber: waiverId ?? "" }}
```

We need a default string because it'll complain about undefined turning into a value